modules/statics/src/coins.ts

```diff
@@ -9448,11 +9448,11 @@ export const coins = CoinMap.fromCoins([
   ),
   erc20(
     'e015ea18-e731-4148-adc1-ab9443459c82',
-    'core',
+    'cvault',
     'cVault.finance',
     18,
     '0x62359ed7505efc61ff1d56fef82158ccaffa23d7',
-    UnderlyingAsset.CORE
+    UnderlyingAsset.CVAULT
```

Comment:
Won't this cause an issue in platform for existing transfers/objects referencing this name?